cli/root.go

```diff
@@ -434,6 +435,14 @@ func (r *RootCmd) Command(subcommands []*clibase.Cmd) (*clibase.Cmd, error) {
 			Value:       clibase.StringOf(&r.globalConfig),
 			Group:       globalGroup,
 		},
+		{
+			Flag:          varOrganizationSelect,
+			FlagShorthand: "z",
+			Env:           "CODER_ORGANIZATION",
+			Description:   "Select which organization (uuid or name) to use as default. This overrides what is present in the config file.",
```

Comment:
```suggestion
			Description:   "Select which organization (uuid or name) to use. This overrides what is present in the config file.",
```

I think the default terminology here could be confusing.